src/Aspire.Hosting/DistributedApplication.cs

```diff
@@ -63,11 +189,46 @@ public static IDistributedApplicationBuilder CreateBuilder(DistributedApplicatio
     /// <summary>
     /// Gets the <see cref="IServiceProvider"/> instance configured for the application.
     /// </summary>
+    /// <remarks>
+    /// <para>
+    /// The <see cref="DistributedApplication"/> is an <see cref="IHost"/> implementation and as such
+    /// exposes a <see cref="Services"/> property which allows developers to get services from the
+    /// dependency injection container after <see cref="DistributedApplication" /> instance has been
+    /// built using the <see cref="IDistributedApplicationBuilder.Build"/> method.
+    /// </para>
+    /// <para>
+    /// To add services to the dependency injection container developers should use the <see cref="IDistributedApplicationBuilder.Services"/>
+    /// property to access the <see cref="IServiceCollection"/> instance.
+    /// </para>
+    /// </remarks>
     public IServiceProvider Services => _host.Services;
 
     /// <summary>
     /// Disposes the distributed application by disposing the <see cref="IHost"/>.
     /// </summary>
+    /// <remarks>
+    /// <para>
+    /// Typically developers do not need to worry about calling the Dispose method on the <see cref="DistributedApplication"/>
```

Comment:
Code-format `Dispose`?

Reply:
For inline code formatting, it's `<c>Dispose</c>`. I agree with this one.